gke/standard/zonal/secondary-boot-disk/main.tf

```
@@ -0,0 +1,44 @@
+/**
+* Copyright 2024 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START gke_standard_secondary_boot_disk]
+resource "google_container_cluster" "cluster" {
+  name                = "default-cluster"
+  location            = "us-central1-a"
+  initial_node_count  = 1
+  deletion_protection = false
+  min_master_version  = "1.28"
```

Would be ideal if this can be dropped for long term maintenance and compatibility.

---

Secondary boot disk is not supported before GKE 1.28.3-gke.106700. Our terraform acceptance test ran into a support error without the min_master_version in https://github.com/GoogleCloudPlatform/magic-modules/pull/10511

---

Got it - Could we append some sort of note to that effect?
```suggestion
  min_master_version  = "1.28" # secondary_boot_disks require GKE 1.28.3-gke.106700 or later
```

---

Added!